models/Asset/Video.php

```diff
@@ -129,6 +129,10 @@ public function getThumbnailConfig($config)
 
         if (is_string($config)) {
             $thumbnail = Video\Thumbnail\Config::getByName($config);
+
+            if($thumbnail === null) {
+                throw new Model\Exception\NotFoundException('Video Thumbnail definition "' . $config . '" does not exist');
```

Comment:
Missing PHPdoc (also for models/Asset/Video/ImageThumbnail.php )

Reply:
Fixed. Thank you.